### PR TITLE
Move the lint from Travis to TaskCluster

### PR DESCRIPTION
**Pull request by jgraham**

### CODE REVIEW

**Comment by foolip** [comment]
@jgraham no Taskcluster check yet, do you know why? Do we need to revert?

**Comment by jgraham** [comment]
Weird, I guess I broke something in the yml file.

File: .taskcluster.yml [hunk 1]
```diff
@@ -166,3 +167,55 @@ tasks:
...
                       --channel=${browser.channel}
                       ${operation.extra_args};
+    - $if: tasks_for == "github-pull-request"
+      # PR tasks that aren't running the changed tests
+      then:
+        $if: event.action in ['opened', 'reopened', 'synchronize']
+        then:
+          $map:
+             - name: lint
+               description: >-
```
Comment (foolip) [inline .taskcluster.yml]: Where does this description end up?

File: .travis.yml [hunk 2]
```diff
@@ -28,11 +28,6 @@ matrix:
           secure: "EljDx50oNpDLs7rzwIv+z1PxIgB5KMnx1W0OQkpNvltR0rBW9g/aQaE+Z/c8M/sPqN1bkvKPybKzGKjb6j9Dw3/EJhah4SskH78r3yMAe2DU/ngxqqjjfXcCc2t5MKxzHAILTAxqScPj2z+lG1jeK1Z+K5hTbSP9lk+AvS0D16w="
         file: $WPT_MANIFEST_FILE.gz
         skip_cleanup: true
-    - name: "lint"
-      # lint is run both on master and on PRs
```
Comment (foolip) [inline .travis.yml]: Shouldn't we keep running lint on master?

**Review by foolip — APPROVED** [review]

File: .taskcluster.yml (hunk 1 above)
Comment (foolip) [reply .taskcluster.yml]: Real question: can we have this as the name in checks? I didn't know there was a separate description before I saw this.

**Review by foolip — APPROVED** [review]

**Comment by gsnedders** [comment]
@jgraham are you sure this isn't broken, given the @taskcluster comment above?

**Comment by foolip** [comment]
Yeah, it was broken. I told @jgraham on IRC, should have said it here :)

**Pull request merged**